README.md

```diff
@@ -17,16 +19,32 @@ Mirecka J, Famili M, Kotanska A, Jurashcko N, Costa-Gomes B, Palmer CM, Thiyagal
 
 > Note: This has been tested in the `refactor` branch.
```

Comment:
This note is no longer relevant - `refactor` branch no longer exists. Either delete the note or change branch to `develop`

Reply:
oh, this is a very old comment, will update to develop.